stm/stm.ml

```diff
@@ -2681,7 +2681,7 @@ let process_transaction ~doc ?(newtip=Stateid.fresh ())
       | VtStartProof (guarantee, names) ->
 
          if not (get_allow_nested_proofs ()) && VCS.proof_nesting () > 0 then
-           "Nested proofs are not allowed unless you turn the Nested Proofs Allowed flag on."
+           "Nested proofs are not allowed unless you turn the Nested Proofs Allowed flag on. This error can also indicate that you forgot to close the last `Proof.` with `Qed.` or `Defined.`"
```

Comment:
Great idea to improve error messages.

This probably should use double quote (`"`) instead of a back-tick or grave (``` ` ```).  Sphinx doesn't interpret back ticks when they appear in error messages, BTW.

Also, beyond the scope of this PR, the error message index is a bit of a mess.  The entries are not wrapped (though you can jump to the text to see the full text) and if I click on "n", it becomes unreadable (there's no way to scroll left).

![image](https://user-images.githubusercontent.com/1253341/100383956-caaef280-2fd3-11eb-9f54-284df5827fc6.png)

![image](https://user-images.githubusercontent.com/1253341/100384067-1eb9d700-2fd4-11eb-8956-c2c854272e55.png)

Reply:
> Also, beyond the scope of this PR, the error message index is a bit of a mess. The entries are not wrapped (though you can jump to the text to see the full text) and if I click on "n", it becomes unreadable (there's no way to scroll left).

Yes, that's worth opening an issue IMHO.